swagger-ui/hooks.go

```diff
@@ -0,0 +1,379 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/snikch/goodman/hooks"
+	trans "github.com/snikch/goodman/transaction"
```

Comment:
This requirement doesn't appear in `go.mod`.

Reply:
ok I will add to the go.mod file